src/mobot_client/drop_runner.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
```

Comment:
@zabrador This file mostly hails from mobot.py:

https://github.com/mobilecoinofficial/mobot/blob/dev/src/mobot_client/mobot.py